packages/plugins/plugin-space/src/components/SpacePluginSettings.tsx

```diff
@@ -27,21 +27,28 @@ export const SpacePluginSettings = ({ settings }: { settings: SpaceSettingsProps
         <Input.Switch checked={settings.showHidden} onCheckedChange={(checked) => (settings.showHidden = !!checked)} />
       </DeprecatedFormInput>
       <div role='none'>
-        <h2 className='text-xl my-4'>Space Settings</h2>
-        <List classNames='max-w-md mx-auto'>
-          {spaces.map((space) => (
-            <ListItem.Root key={space.id}>
-              <ListItem.Heading classNames='flex flex-col grow truncate mbe-2'>
-                {toLocalizedString(getSpaceDisplayName(space, { personal: space === client.spaces.default }), t)}
-              </ListItem.Heading>
-              <ListItem.Endcap>
-                <Button onClick={() => dispatch(createIntent(SpaceAction.OpenSettings, { space }))}>
-                  {t('open space settings label')}
-                </Button>
-              </ListItem.Endcap>
-            </ListItem.Root>
-          ))}
-        </List>
+        <h2 className='text-xl my-4'>{t('space settings label')}</h2>
+        {/* TODO(burdon): Consider fragment for a callout border section. */}
+        <div className='-mis-4 -mie-4 p-4 border border-separator rounded-md'>
+          <List classNames='flex flex-col gap-2'>
+            {spaces.map((space) => (
+              <ListItem.Root key={space.id} classNames='items-center gap-4'>
+                {/* TODO(burdon): Should auto center and truncate; NOTE truncate doesn't work with flex grow. */}
+                <ListItem.Heading classNames='flex w-full items-center truncate'>
+                  {toLocalizedString(getSpaceDisplayName(space, { personal: space === client.spaces.default }), t)}
+                </ListItem.Heading>
+                <ListItem.Endcap>
+                  <IconButton
+                    icon='ph--faders--regular'
+                    iconOnly
```

Comment:
```suggestion
                    
```
This is not the place to omit labels, there is plenty of room so this is unnecessarily enigmatic.